app/controllers/conference_registrations_controller.rb

```diff
@@ -40,6 +41,7 @@ def show
   def edit; end
 
   def create
+    authorize! :create, Registration.new(conference_id: @conference.id)
     @registration = @conference.registrations.new(registration_params)
```

Comment:
hey, doesn't validation of registration for [presence of user](https://github.com/openSUSE/osem/blob/master/app/models/registration.rb#L26) cover us here?